elftools/elf/sections.py

```diff
@@ -19,16 +22,83 @@ class Section(object):
          > sec = Section(...)
          > sec['sh_type']  # section type
     """
-    def __init__(self, header, name, stream):
+    def __init__(self, header, name, stream, structs):
```

Comment:
I went even further, removing the `stream` constructor argument, which can be replaced by `self.ellfile.stream`. I hope that’s fine by you.